packages/aws-cdk/test/integ/cli/aws-helpers.ts

```diff
@@ -138,6 +154,52 @@ export async function retry<A>(operation: string, deadline: Date, block: () => P
   }
 }
 
+/**
+ * Make a deadline for the `retry` function relative to the current time.
+ */
+retry.forSeconds = (seconds: number): Date => {
+  return new Date(Date.now() + seconds * 1000);
+};
+
+/**
+ * Annotate an error to stop the retrying
+ */
+retry.abort = (e: Error): Error => {
```

Comment:
I feel like the names here are a little weird, that's why I couldn't understand this code at first... `abort` means "stop and bail out!", not "mark something as aborted"... how about something like `markAsAborted` instead?